web/mobile_style.html

```diff
@@ -28,6 +28,30 @@
     --paper-toggle-button-unchecked-ink-color: var(--text-primary-color);
   }
 
+  #errorToast {
```

Comment:
did you mean to make these changes to mobile_style.html as part of the embedding PR? Are they addressing issues?